docs/source/en/perf_infer_gpu_one.md

```diff
@@ -17,6 +17,154 @@ rendered properly in your Markdown viewer.
 
 In addition to this guide, relevant information can be found as well in [the guide for training on a single GPU](perf_train_gpu_one) and [the guide for inference on CPUs](perf_infer_cpu).
 
+## Flash Attention 2
+
+<Tip>
+
+Note that this feature is experimental and might considerably change in future versions. For instance, the Flash Attention 2 API might migrate to `BetterTransformer` API in the near future.
+
```

Comment:
> the Flash Attention 2 API might migrate to `BetterTransformer` API in the near future. 

did not know this was planned 😄 If not let's just not say anything

Reply:
Well it is all about providing a single meaningful API to users and avoid confusing them. In PyTorch 2.2 (hopefully not too late!), we'll be in a state where FA2 will be supported by SDPA so basically a duplicate of this.

Reply:
this has been discussed internally and it was the consensus we have all found together